app.rb

```diff
@@ -0,0 +1,114 @@
+module App
+  INPT_MSG = 'Enter an option number here --> '.freeze
+  ENTR_MSG = 'Press ENTER to continue'.freeze
+
+  def enter_msg
+    print ENTR_MSG
+    gets
+    puts
+  end
+
+  def display_books
+    puts 'List of books:'
+    puts 'EMPTY' if @books.empty?
+    @books.each_with_index { |book, index| puts "#{index}) Title: #{book.title}, Author: #{book.author}" }
+  end
+
+  def display_people
+    puts 'List of people:'
+    puts 'EMPTY' if @people.empty?
+    @people.each_with_index do |person, index|
+      puts "#{index}) Name: #{person.name}, Age: #{person.age}, ID: #{person.id}"
+    end
+  end
+
+  # rubocop:disable Metrics
+  def create_person
+    decision = 0
+    until [1, 2].include?(decision)
+      puts 'Do you want to create a student (1), or a teacher (2)?'
+      print INPT_MSG
+      decision = gets.chomp.to_i
+      next if [1, 2].include?(decision)
+
+      puts
+      puts 'Invalid input, please try again'
+      puts
+    end
+    print 'Age --> '
+    age = gets.chomp.to_i
+    print 'Name --> '
+    name = gets.chomp
+    case decision
+    when 1
+      create_student(age, name)
+    else
+      create_teacher(age, name)
+    end
+    puts
+  end
+
+  # rubocop:enable Metrics
+
+  def create_book
+    puts 'Please, enter book information below:'
+    print 'Title --> '
+    title = gets.chomp
+    print 'Author --> '
+    author = gets.chomp
+    @books << Book.new(title, author)
+    puts
+    puts 'Book created successfully'
+    puts
+  end
```

Comment:
Same as previous comment. :arrow_up:
`create_book` method should only contain line 59